oat-sa/extension-tao-itemqti
# feat/AUT-964/ e2e Item Authoring, Assign correct answer

Related to: https://oat-sa.atlassian.net/browse/AUT-964

Covered by e2e testing page item Authoring. Assigned correct answers (where possible)

**How to test:**
- switch tao to branch https://github.com/oat-sa/tao-core/pull/3147
- run cypress from tao/views 
- check test `taoQtiItem/views/cypress/tests/item-authoring-set-response.spec.js`
